tests/spec/SpecHelper.js

```diff
@@ -57,6 +57,78 @@ export function makeRSDoc(opts, src, style = "") {
   });
 }
 
+/**
+ * Create a doc for unit tests.
```

Comment:
This is neat! I like the idea of running things in isolation. However, we should use this sparingly, as running all the interdependencies always catches many bugs in other code.

Reply:
I'm thinking of creating a separate folder for unit tests, which would run before current (integration) tests. Might move some of the integration tests into unit tests, where it makes sense.
Agree on using unit tests sparingly (keeping in mind unit tests are unit tests!)

Reply:
I like the division of labor. The utils module does something similar, in as far as we test exported methods on their own.... I like where this is going!